test/webhook_test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+kubectl apply -f https://github.com/FairwindsOps/polaris/releases/latest/download/webhook.yaml
```

Comment:
This should be `./deploy/webhook.yaml`, so we test against the latest deploy configuration.

Let's also add this line before `kubectl apply`:

`sed -ri "s|'(quay.io/reactiveops/polaris:).+'|'\1${CIRCLE_SHA1}'|" ./deploy/webhook.yaml`

You can see a similar line just got added to the `test_kube_dashboard` job. This changes `deploy/webhook.yaml` to use the code that's being checked in, rather than the last release.